di-ipv-orchestrator-stub/build.gradle

```diff
@@ -1,4 +1,5 @@
 plugins {
+	id 'com.github.johnrengelman.shadow' version '8.1.1'
```

Comment:
This was used to generate a fat jar.  If there's a more appropriate way, please do change this.